lifecycle/README.rst

```diff
@@ -63,19 +59,19 @@ We split the tasks of the talker node into separate pieces and execute them as f
 #. cleanup: We destroy the publisher and timer
 
 The principle is implemented in this demo as the typical talker/listener demo.
```

Comment:
What principle? Instead of mentioning a principle that wasn't introduced could simply say that the demo implements a lifecycle node.